register.html

```diff
@@ -25,10 +25,29 @@ <h1 class="logo">Relaxr</h1>
 				</nav>
 			</div>
 		</header>
-		<section id="main">
+		<section class="yellow" id="main">
 			<div class="container">
 				<section id="contact">
-					<input type="submit" value="Register Now!">
+					<label>Username</label>
+					<input type="text" name="username" Placeholder="Joe">
+					<label>Password</label>
+					<input type="password" name="password">
+					<label>Occupation</label>
+					<select name="Type of Occupation">
+						<option value="Blue Collar">Blue Collar</option>
+						<option value="White Collar">White Collar</option>
+						<option value="No Collar">No Collar</option>		
+					</select>
+					<br>
```

Comment:
Avoid the use of `br` tags - it's preferable to use wrapper elements to achieve breaks between elements. Maybe wrap each pair of input/label elements in a div?